pyproject.toml

```diff
@@ -1,5 +1,6 @@
 [build-system]
-requires = ["setuptools < 67.0.0", "wheel"]
+requires = ["setuptools"]
```

Comment:
According to Git, you were pinning to `<67` to allow using pre-releases in requirements.txt. You aren't using those anymore.

`wheel` used to be incorrectly in setuptools' docs, but they've fixed it to say you shouldn't include here.